### DISCUSSION TITLE
Issues ordering from JLCPCB

### DISCUSSION THREAD

**Discussion by moarkaboom** [CrashOverride85/zc95#118] [Q&A] [answered]
Hello all,

So after finding this awesome project, obviously I want to build my own. At the moment I am getting my JLCPCB order ready, but it seems the online tool is pretty finicky. 
When choosing the output board, I get this message: 
![missing](https://github.com/user-attachments/assets/01d0358d-eed1-4b84-a76b-1578032729f6)

Is that because these parts will be hand soldered or is something else wrong?

Also, when following the guide for the PanelControls, there seems to be a typo. The files are correct, just the links are named wrong:
![image](https://github.com/user-attachments/assets/31fbf119-1a0c-4743-be44-e21185b49aa3)

I am pretty new to the whole electronics thing and just want to make sure I order everything correctly so I dont end up with expensive paper weights.

Oh, and is there a Discord for this project?

**Comment by CrashOverride85** [comment, marked as answer]
Yes, that message is normal as those are hand assembly parts.
J1/4 are the 2.54mm headers, TR1-4 the transformers and U4 the Pico-W.
And good spot, now fixed :)

It's probably only fair of me to point out that if you're new to electronics, this isn't an _ideal_ first project - there's quite a few steps and there's probably some amount of assumed knowledge / experience in the build unfortunately.

There isn't a discord for this project, but I'd recommend joining [Joanne's E-Stim Community](https://discord.gg/rY8C27S) discord server, of particular interest is the #boxes-pulse-based-diy channel.

**Reply by moarkaboom** [reply]
Thanks for clearing that up. I am not entirely new to electronics, my skills are just very basic. I have done a few arduino based projects and have built three different 3D printers. so after I read your guide I was certain it was within my skillset. And luckily for me, you are very responsive and help out, so thank you for that!

I am a member of Joanne's Discord but when I searched for ZC-95 or ZC95, nothing showed up.

**Reply by CrashOverride85** [reply]
Oh ok, sounds like you shouldn't have any problems then :) 

There's been a handful messages relating to the zc95 over the last year or two, but not a massive about

**Reply by electro991** [reply]
@moarkaboom i have built a few units and would be happy to help if you have any questions too. I would say that the parts you need to be the most careful with based on my experience are the 2 4-pin voltage regulators on the main board and the 6 leds on the front panel. The pads are really really close and they’re easy to short. Check existing shorts before soldering as 2 side by side pads on one of them are short by design.  After my first build, i started doing those and the leds with chipquik (138) and hot air at 240  instead of the traditional iron. Found it much easier.

**Reply by moarkaboom** [reply]
@electro991 Thanks for your comments and the offer to help! I will be sure to post here, if I run into problems. At the moment everything is ordered and I am just waiting for components to arrive.
